server/modules/user/user.service.ts

```
@@ -196,6 +198,54 @@ class UserService {
+      })
+      .lean();
+
+      // Extract organizations and filter out any undefined/null values
+      const organizations = userRoleAssignments
+        .filter(assignment => assignment.scopeId !== null && assignment.scopeId !== undefined)
+        .map(assignment => assignment.scopeId);
+
+      // Remove duplicate organizations using a Map with string IDs as keys
+      const uniqueOrganizationsMap = new Map();
+      organizations.forEach(org => {
+        if (org && org._id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)